src/components/Editor/SearchBar.js

```diff
@@ -437,27 +448,41 @@ const SearchBar = React.createClass({
   },
 
   renderSearchTypeToggle() {
-    if (!isEnabled("functionSearch")) {
+    if (!isEnabled("symbolSearch")) {
       return;
     }
+    const { toggleSymbolSearch } = this;
+    const { symbolSearchEnabled, selectedSymbolType } = this.state;
+
+    function searchTypeBtn(searchType) {
+      return dom.button({
+        className: classnames("search-type-btn", {
+          active: symbolSearchEnabled && selectedSymbolType == searchType
+        }),
+        onClick: e => {
+          if (selectedSymbolType == searchType) {
```

Comment:
we should re-trigger a search when there's a query. 

STR:
1. type `t`
2. select function
3. select variables should re-search with `t` or clear the query